testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -245,6 +278,31 @@ def test_Writer(self, tmpdir):
             u.atoms.positions, universe.atoms.positions, self.prec
         )
 
+    def test_Writer_with_dt_setting(self, tmpdir):
+        universe = mda.Universe(GRO, self.filename, convert_units=True)
```

Comment:
I don't like replicating the code of `test_Writer`, less copy&paste makes for more maintainable code. Can you please refactor so that the two methods share most of their code? Perhaps there's even a way to make it a parametrized test.